src/tests/__snapshots__/dotIgnore.test.ts.snap

```diff
@@ -22,7 +22,6 @@ exports[`dot ignore merges the lines 1`] = `
 
         # another comment
         .idea
-
```

Comment:
This is a change to the snapshot?

Reply:
It's because you had an extra element in the array: https://github.com/wholesomedev/extend-config/pull/1/files/28dde366ce20f79866a22bbeba3cf7fef9c2d966#diff-a7f42f15ab86caf28e38dc63b3064fe9L6

```
const combinedLines = [...sourceLines, , ...destinationLines]
```

Was that intended?

Reply:
No it wasn't! :D

I'll review the snapshots again to make sure they make sense.